### ISSUE TITLE
i need to apply the key after a reboot

### ISSUE THREAD

**Issue by alavaelian** [sezanzeb/input-remapper#432]
i looking a way to make the remap permanent
but when i reboot the computer it not work until i open the program again and click the apply button

**Comment by alavaelian** [comment]
no, Thanks mate that worked

**Comment by sezanzeb** [comment]
You didn't provide any logs unfortunately, so I didn't look into it.

Is this still an issue?

**Comment by sam-kleiner** [comment]
This is an issue for me as well on ubuntu 22.04 default desktop env. I thought maybe it was trying to start before my mouse connected so I added an after bluetooth target as well as a 15 second delay to the systemd command. No change with that

```
input-remapper-control --version
input-remapper 1.4.0  https://github.com/sezanzeb/input-remapper
python-evdev 1.4.0

```

```
cat ~/.config/input-remapper/config.json
{
    "version": "1.4.0",
    "autoload": {
        "Logitech Wireless Mouse MX Master 3": "Global"
    },
    "macros": {
        "keystroke_sleep_ms": 10
    },
    "gamepad": {
        "joystick": {
            "non_linearity": 4,
            "pointer_speed": 80,
            "left_purpose": "none",
            "right_purpose": "none",
            "x_scroll_speed": 2,
            "y_scroll_speed": 0.5
        }
    }
}
```

```
cat /home/sam/.config/input-remapper/presets/Logitech\ Wireless\ Mouse\ MX\ Master\ 3/Global.json 
{
    "mapping": {
        "1,276,1": [
            "Control_L + Alt_L + KEY_LEFT",
            "keyboard"
        ],
        "1,275,1": [
            "Control_L + Alt_L + KEY_RIGHT",
            "keyboard"
        ]
    }
}
```
After a fresh boot
```
systemctl status input-remapper
● input-remapper-daemon.service - Service to inject keycodes without the GUI application
     Loaded: loaded (/lib/systemd/system/input-remapper-daemon.service; enabled; vendor preset: enabled)
     Active: active (running) since Sun 2022-09-18 07:09:08 EDT; 26s ago
    Process: 3716 ExecStartPre=/bin/sleep 15 (code=exited, status=0/SUCCESS)
   Main PID: 6552 (input-remapper-)
      Tasks: 4 (limit: 76918)
     Memory: 24.2M
        CPU: 160ms
     CGroup: /system.slice/input-remapper-daemon.service
             ├─6552 /usr/bin/python3 /usr/bin/input-remapper-service
             └─6553 /usr/bin/python3 /usr/bin/input-remapper-service

Sep 18 07:08:52 sam-desktop systemd[1]: Starting Service to inject keycodes without the GUI application...
Sep 18 07:09:07 sam-desktop input-remapper-service[6552]: Starting logging to "/var/log/input-remapper"
Sep 18 07:09:07 sam-desktop input-remapper-service[6552]: input-remapper-service 1.4.0  https://github.com/sezanzeb/input-remapper
Sep 18 07:09:07 sam-desktop input-remapper-service[6552]: python-evdev 1.4.0
Sep 18 07:09:08 sam-desktop systemd[1]: Started Service to inject keycodes without the GUI application.

sam@sam-desktop:~$ cat /var/log/input-remapper
Starting logging to "/var/log/input-remapper"
input-remapper-service 1.4.0  https://github.com/sezanzeb/input-remapper
python-evdev 1.4.0
```
After opening the app and hitting apply
```
cat /var/log/input-remapper
Starting logging to "/var/log/input-remapper"
input-remapper-service 1.4.0  https://github.com/sezanzeb/input-remapper
python-evdev 1.4.0
Loaded config from "/home/sam/.config/input-remapper/config.json"
Loaded config from "/home/sam/.config/input-remapper/config.json"
Found "input-remapper mouse", "Logitech Wireless Mouse MX Master 3", "USB Keyboard", "ACTIONS Pebble V3"
Loading preset from "/home/sam/.config/input-remapper/presets/Logitech Wireless Mouse MX Master 3/Global.json"
Starting injecting the mapping for "Logitech Wireless Mouse MX Master 3"
```

**Comment by sezanzeb** [comment]
First, please install the newest .deb release from https://github.com/sezanzeb/input-remapper/releases/tag/1.5.0 and see if this has already been fixed

**Comment by DamienHodsall** [comment]
I have been having this issue too. I am mx linux which uses sysvinit by default. When I tried to check the status I got an error saying 'Failed to enable unit, unit key-mapper.service does not exist.' I also get an error when I run input-remapper-gtk saying 'System has not been booted with systemd as init system (PID 1). Can't operate.
Failed to connect to bus: Host is down'. I finally tried installing the latest .deb package and got another error saying 'dpkg-deb: error: archive 'input-remapper-1.5.0.deb' uses unknown compression for member 'control.tar.zst', giving up'. The full outputs are attached. Thanks.

[remapper-output.txt](https://github.com/sezanzeb/input-remapper/files/9640416/remapper-output.txt)
